Source/DZNSegmentedControl.h

```diff
@@ -135,3 +137,9 @@
  */
 @protocol DZNSegmentedControlDelegate <UIBarPositioningDelegate>
 @end
+
+@protocol DZNSegmentedControlSegmentDelegate
```

Comment:
Why not add this to the protocol, above?

Reply:
The intended use of `DZNSegmentedControlDelegate` seemed to be about bar positioning, but in retrospect if that was the case then there's no reason to not use straight up `UIBarPositioningDelegate`. I'll move it into there.